sway-book/src/documentation/examples/subcurrency.md

```diff
@@ -0,0 +1 @@
+# Subcurrency
```

Comment:
Please, can we do away with the subcurrency example, unless we supplement it with a fuel native token example?
For new users, the subcurrency example makes it seem like ledger-based tokens are the recommended way to implement a token contract on forc.